db/mongo/client.go

```diff
@@ -15,7 +17,7 @@ import (
 )
 
 const (
-	dbName        = "ApnaSchool"
+	dbName        = viper.GetString(configuration.DbName)
```

Comment:
we are not going to use `const` anymore. use `viper.GetString(configuration.DbName)` directly where it's needed.